source/blog/2016-12-14-size-and-change.html.md

```diff
@@ -0,0 +1,165 @@
+not do well.
+
+## Structure and Relationships
+
+At each of these changes, required structure and process is one axis.
+The other axis, perhaps more significant, is how people in the group
+relate to each other. Individuals can have multiple identities at once,
+each of which is stronger or weaker. I can be part of a project and part
+of a company at the same time&mdash;and whether I consider myself an project
+foo developer who gets paid by company X, or a company X employee who
+happens to be working on project foo, is a very personal thing,
+depending on the depth of the relationships that I have with other
+members of the groups "project foo" and "company X."
+
+The relationships with group members will change over time, as group
+size changes. I started my developer career as a developer of the GIMP,
+spent over a year as the GIMP release manager, and organized multiple
+GIMP events, but over time I have drifted away as the project evolved,
+and some of the people I had close relationships with in the project
+reduced their activity. At the same time as my ties to the GIMP were
+waning, I was spending more and more time in the GNOME project&mdash;and
+around 2004 or 2005, I would have considered myself more a part of the
+GNOME project than the GIMP. And so on throughout my career. I have
+maintained some lifelong friendships with current or former GNOME
+project members, but that is not such a big part of my identity any more.
+
+There are a number of ways that identity can change as groups grow&mdash;and
+one of them is to break into sub-groups. A Nova developer may still feel
+like an OpenStack developer, but the personal connections with people
+working on the Nova project will be stronger. And the Nova core
+reviewers group will have even stronger relationships with each other.
+You see this happen with conferences all the time. When a conference
+starts out, and the number of attendees is in the low hundreds, you have
+a small number of organizers who are close friends, and attendees have
+rich relationships and conversations. As a conference gets to the high
+hundreds, you start to see "tracks" form, where big sub-groups gather to
+share knowledge specific to them&mdash;with a resulting lessening of
+awareness of what is happening in other projects. When the OpenStack
+Summit got towards 1500 or 2000 people, you start to see a completely
+separate sub-event forming for developers&mdash;the "OpenStack developer"
+identity is reinforced, at the expense of some awareness of the
+technical community in the greater OpenStack ecosystem. And now, with
+the main event over 5000 attendees, another inflection point has been
+reached, where in addition to per-project tracks at the developer
+summit, specific projects are co-ordinating smaller "mid-stream" events
+to encourage the creation of an even tighter per-project active
+participant identity. Once again those numbers&mdash;150, 500, 1500, 5000 -
```

Comment:
Addition: "From next year, the developer event will be held completely separately, which will help to reinforce the "OpenStack developer" identity by making that event smaller."